compiler_opt/rl/train_locally.py

```diff
@@ -100,23 +99,14 @@ def train_eval(agent_name=constant.AgentName.PPO,
   }
   saver = policy_saver.PolicySaver(policy_dict=policy_dict)
 
-  with open(
-      os.path.join(FLAGS.data_path, 'module_paths'), 'r',
-      encoding='utf-8') as f:
-    module_paths = [
-        os.path.join(FLAGS.data_path, name.rstrip('\n')) for name in f
-    ]
-
-    if not problem_configuration.is_thinlto(module_paths):
-      file_paths = [(path + '.bc', path + '.cmd') for path in module_paths]
-    else:
-      file_paths = [(path + '.bc', path + '.cmd', path + '.thinlto.bc')
-                    for path in module_paths]
+  logging.info('Loading module specs from corpus')
+  module_specs = problem_config.get_spec_type().get(FLAGS.data_path,
```

Comment:
I think you can further decouple the spec thing from the corpus and problem solvers and so on: just let it be a problem-internal consideration.

Reply:
currently it seems to be a problem-internal consideration and decoupled well, how the further decoupling looks like?

Reply:
they're back in `*_runner` now